models/project/project.go

```diff
@@ -61,6 +62,89 @@ func FindProject(projectID int64) *ProjectStruct {
 	}
 }
 
+func Create(userID int64, title string, description string, repositoryID int64, repositoryOwner string, repositoryName string, oauthToken sql.NullString) (p *ProjectStruct, e error) {
```

Comment:
__[golint]__
* [golint] exported function Create should have comment or be unexported